freqtrade/plugins/protections/cooldown_period.py

```diff
@@ -18,7 +18,10 @@ def _reason(self) -> str:
         """
         LockReason to use
         """
-        return f"Cooldown period for {self.stop_duration_str}."
+        reason = f"Cooldown period for {self.stop_duration_str}."
+        if self.unlock_at_str is not None:
+            reason += f" Unlocking trading at {self.unlock_at_str}."
+        return reason
```

Comment:
You'll want to be more clever with this for sure.

Currently, it'll output something like:
"Cooldown period for 60 minutes. Unlocking trade at 12:00"  (which is wrong - it's either 60 minutes, OR 12:00).

(60 is the default for _stop_duration - which will most likely apply here).
Obviously, this applies to all protections

you might wanna change this a bit - to not use `self.stop_duration_str` directly, but some pre-formatted string from IProtections to determine the "unlock time" in a better way